stixpy/vis/aia_reprojection.py

```diff
@@ -0,0 +1,235 @@
+from datetime import datetime
+from stixcore.ephemeris.manager import Position
+import stixcore.data.test
+from pathlib import Path
+import sunpy
+import sys
+from sunpy.coordinates import get_body_heliographic_stonyhurst
+from astropy.coordinates import SkyCoord
+import astropy.units as u
+from sunpy.map import Map
+from sunpy.coordinates import get_body_heliographic_stonyhurst
+from sunpy.coordinates import frames
+from astropy.wcs import WCS
+from pathlib import Path
+from reproject import reproject_interp
+import matplotlib.pyplot as plt
+from sunpy.net import Fido
+from sunpy.net import attrs as a
+plt.rcParams.update({'font.size': 7})
+
+def get_SOLO_Pos(start_day, end_day):
+    """
+        Return the position of SOLO in SOLO_HEE coordinate frame
+        Parameters
+        ----------
+        start_day : string
+            Provide a string of the start date with YYYY-DD-MM formatting. e.g '2020-03-09'
+        end_day : string
+            Provide a string of a the end date with YYYY-DD-MM formatting. e.g '2020-03-10'
+        Returns
+        -------
+        p : tuple
+            position of SOLO in SOLO_HEE coordinate frame
+    """
+    # Specifiying year, month, day
+    start_year = int(start_day[0:4])
+    start_month = int(start_day[5:7])
+    start_day = int(start_day[8:10])
+
+    from pathlib import Path
+    mkp = Path(stixcore.ephemeris.manager.__file__).parent.parent / 'data' / 'test' / 'ephemeris' / 'test_position_20201001_V01.mk'
+    with Position(meta_kernel_path=mkp) as pos:
+        p = pos.get_position(date=datetime(start_year, start_month, start_day), frame='SOLO_HEE')
+    print(p)
+    return(p)
+#__________________________________
+
+
+def convert_SOLO_coords(start_day, end_day, p, aiamap):
+    """
+        Returns the position of SOLO in HeliographicStonyhurst frame and returns the reference frame (AIA as seen by SOLAR ORBITER)
```

Comment:
The formatting just needs a bit of a tweak all the lines should start with 4 space and leave a new line between the sections